core/modules/wdb/QueryRunner.cc

```diff
@@ -398,11 +401,23 @@ bool QueryRunner::_dispatchChannel() {
                 break;
             }
             proto::TaskMsg_Fragment const& fragment(m.fragment(i));
+            std::vector<std::string> queries;
+            for (const std::string queryStr: fragment.query()) {
+                if (fragment.has_subchunks() && false == fragment.subchunks().id().empty()) {
+                    for (auto subchunkId : fragment.subchunks().id()) {
+                        std::string s(queryStr);
+                        boost::algorithm::replace_all(s, SUBCHUNK_TAG, std::to_string(subchunkId));
```

Comment:
This looks like it does the same thing as replace(std::string const& s, std::string const& pat, std::string const& value) in qana/QueryMapping.cc. We should probably use the same function in both places.